cves/CVE-2019-14235.yml

```diff
@@ -234,37 +226,37 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: If a user finds a way to crash with the certain inputs then they easily exploit it by doing the inputs constantly to create denial of service.
   security_by_obscurity:
-    applies:
+    applies: false
     note:
   serial_killer:
-    applies:
+    applies: false
     note:
   environment_variables:
-    applies:
+    applies: false
     note:
   secure_by_default:
-    applies:
-    note:
+    applies: true
+    note: This is a working method, there are as the CVE says certain inputs that will crash the system
```

Comment:
I'm confused by this answer. How does it relate to secure by default? What default exists in this method that should be secure?